<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTableView" name="volumesView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #e0e0e0;</string>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="FilesTableView" name="filesView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="errorArea" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 106, 106);
color: rgb(255, 255, 255);</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="errorLabel">
             <property name="text">
              <string notr="true">This is error text</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonCloseErrorArea">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionNext"/>
   <addaction name="actionUp"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionHome"/>
   <addaction name="separator"/>
   <addaction name="actionUpload"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionTasks"/>
  </widget>
  <action name="actionBack">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/remoteBrowser/prev_normal_on.png</normaloff>:/remoteBrowser/prev_normal_on.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/remoteBrowser/next_normal_on.png</normaloff>:/remoteBrowser/next_normal_on.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/remoteBrowser/up_normal_on.png</normaloff>:/remoteBrowser/up_normal_on.png</iconset>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
   <property name="toolTip">
    <string>Up</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/remoteBrowser/refresh_normal_on.png</normaloff>:/remoteBrowser/refresh_normal_on.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/remoteBrowser/home_normal_on.png</normaloff>:/remoteBrowser/home_normal_on.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Home</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/remoteBrowser/upload_normal_on.png</normaloff>:/remoteBrowser/upload_normal_on.png</iconset>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
   <property name="toolTip">
    <string>Upload</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/remoteBrowser/settings_normal_on.png</normaloff>:/remoteBrowser/settings_normal_on.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionTasks">
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/remoteBrowser/tasks_normal_on.png</normaloff>:/remoteBrowser/tasks_normal_on.png</iconset>
   </property>
   <property name="text">
    <string>Tasks</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilesTableView</class>
   <extends>QTableView</extends>
   <header location="global">filestableview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
